tests/sanity_helpers.py

```diff
@@ -132,12 +132,7 @@ def verify_obc(self):
         we will check 2 OBCs
 
         """
-        sample = TimeoutSampler(
-            300,
-            5,
-            self.ceph_cluster.noobaa_health_check
-        )
-        sample.wait_for_func_status(True)
+        self.ceph_cluster.wait_for_noobaa_health_ok()
```

Comment:
Cool, was curious why we don't need sampler anymore here and see that under wait_for_noobaa_health_ok we already have retry and it runs other function which in case it not OK throws NoobaaHealthException , so if we will go over all retries and if I am understanding correctly it will fail with the above mentioned exception, right?

Reply:
100% correct :)